paper.tex

```diff
@@ -35,6 +35,29 @@
 \begin{document}
 \maketitle
 
+\abstract{
+  Symbolic based computer aided algebra systems (CAS) play an important role in
```

Comment:
Ok, It should be: CAS = computer algebra system, but is "symbolic" redundant then?